recipes/morphometrics/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "morphometrics" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/morphometrics-{{ version }}.tar.gz
+  sha256: 096ed657ef55aa1ff61e1fedfcde4ab882fcf115dbe488944da27194bc8e652a
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools-scm
+  run:
+    - cython
+    - hdbscan
+    - jinja2 >=2.10,<3.1
+    - leidenalg
+    - napari
+    - napari-clusters-plotter
+    - napari-skimage-regionprops
+    - numba >=0.55.1
+    - numpy >=1.21
+    - ocl_icd_wrapper_apple  # [osx]
```

Thanks for the tip, @goanpeca !  I added the linux dependency as well.

```suggestion
    - ocl-icd-system # [linux]
    - ocl_icd_wrapper_apple  # [osx]
```